project/profiles/capacity4more/modules/c4m/user/c4m_user_admin/c4m_user_admin.pages.inc

```diff
@@ -39,6 +67,22 @@ function c4m_user_admin_user_activate_form_submit($form, &$form_state) {
 }
 
 /**
+ * Form submission handler for c4m_user_admin_user_activate_form().
+ *
+ * @see c4m_user_admin_user_activate_form()
+ */
+function c4m_user_admin_user_reactivate_form_submit($form, &$form_state) {
+  c4m_user_admin_user_activate_form_submit($form, $form_state);
+
+  if ($form_state['values']['c4m_notify_user']){
+    // Send reactivate notification.
+    $account = $form_state['user'];
+    $params['account'] = $account;
+    drupal_mail('user_registrationpassword', 'register', $account->mail, user_preferred_language($account), $params);
```

Comment:
@ordavidil 
We send here a mail using old mailing system.
Mail subject and body are located within txt files at /capacity4more/modules/c4m/mail/c4m_mail/messages.
_c4m_user_send_mail() expects to receive subject and body as params.
It's also defined at CFM 1200, which is not merged yet.